utils/utils.py

```diff
@@ -271,6 +271,12 @@ def timer_ms():
     return round(timer() * 1000)
 
 
+def remove_fields_from_dict(dictonary, fields_to_remove):
+    for key in fields_to_remove:
```

Comment:
```suggestion
    for key in (fields_to_remove or ()):
```
this way it would accept zero-length `fields_to_remove` (and even `None`), so no need to check above if fields are non-empty, call it unconditionally